### PR TITLE
fix: Failed to start language server: popup is too generic

### PR DESCRIPTION
**Pull request by marianaalanis93**
This fix adds a button to see the output (it redirects you to the output section inside VSCode)  to know the problem when restarting the server. Issue #181 
Before:

image

So I have two possible solutions:
   1.- Show a button to activate output section inside VSCode (included in this PR)
image

   2.- Automatically activate outputChannel (NOT included in this PR):
i.e.
By default, in Azure section:
image

But after detecting the failure (Going to Output section and displaying the error message): 
image

@Kha and @lovettchris let me know your thoughts.

*UPDATE: This PR contains 2nd solution.*

Final solution : 

After checking CR comments, the output message stays like this:
image
Based on: https://github.com/leanprover/vscode-lean4/pull/229

### CODE REVIEW

**Review by lovettchris — APPROVED** [review]

**Review by Kha — APPROVED** [review]

File: vscode-lean4/src/utils/leanInstaller.ts [hunk 1]
```diff
@@ -161,6 +161,7 @@ export class LeanInstaller implements Disposable {
         }
 
         this.promptingInstallEmitter.fire(uri);
+        this.outputChannel.show(true);
```
Comment (Vtec234) [inline vscode-lean4/src/utils/leanInstaller.ts:164]: Are we certain this is the right change? It does not seem to introduce the 'Go to output' button.

Comment (Vtec234) [reply vscode-lean4/src/utils/leanInstaller.ts:164]: Oh, I missed the update at the bottom of the PR description.

Comment (lovettchris) [reply vscode-lean4/src/utils/leanInstaller.ts:164]: I'm also confused now, what happened to the show output button idea?  and show does this new outputChannel.show call relate to the new user configurable `lean4.autofocusOutput` option?  https://github.com/leanprover/vscode-lean4/pull/229

**Comment by marianaalanis93** [comment]
After checking CR comments, the output message stays like this: 
image
Based on: https://github.com/leanprover/vscode-lean4/pull/229

File: vscode-lean4/src/utils/leanInstaller.ts (hunk 1 above)
Comment (Kha) [reply vscode-lean4/src/utils/leanInstaller.ts]: Start failure messages should always be of interest to the user, so I don't think #229 applies here. But an explicit button isn't that bad either.

**Review by Vtec234 — APPROVED** [review]

**Pull request merged**